tests/e2e/framework/kubernetes.go

```diff
@@ -144,42 +144,57 @@ func (k *KubeInfo) Teardown() error {
 	glog.Info("Cleaning up kubeInfo")
 	var err error
```

Comment:
Move `var err error` down to line 183 where its used and return `nil` one line 148.

Reply:
ok